features/step_definitions/command_steps.rb

```diff
@@ -358,6 +361,46 @@
   sle11 = "#{os_version[0, 2]}-SP#{os_version[-1]}"
   cmd = "mgr-create-bootstrap-repo -c SLE-#{os_version}-#{arch}" if os_version.include? '12'
   cmd = "mgr-create-bootstrap-repo -c SLE-#{sle11}-#{arch}" if os_version.include? '11'
-  puts 'Creating the boostrap-repo on the server: ' + cmd
+  puts 'Creating the boostrap repository on the server: ' + cmd
   $server.run(cmd, false)
 end
+
+When(/^I copy server\'s keys to the proxy$/) do
+  ['RHN-ORG-PRIVATE-SSL-KEY', 'RHN-ORG-TRUSTED-SSL-CERT', 'rhn-ca-openssl.cnf'].each do |file|
+    rc = file_extract($server, '/root/ssl-build/' + file, '/tmp/' + file)
+    raise 'File extraction failed' unless rc.zero?
+    $proxy.run('mkdir -p /root/ssl-build')
+    rc = file_inject($proxy, '/tmp/' + file, '/root/ssl-build/' + file)
+    raise 'File injection failed' unless rc.zero?
+  end
+end
+
+When(/^I configure the proxy$/) do
+  # prepare the settings file
+  settings = "RHN_PARENT=#{$server_ip}\n" \
+             "HTTP_PROXY=''\n" \
+             "VERSION=3.2\n" \
```

Comment:
we need to take the version var dinamically from a file, otherwise this is a risk.

Reply:
We have a different branch for each version. The only risk is to forget to bump this version number.

But I'll do it to please you.

Reply:
We can let this,  and if we forget to bump, you pay me 1000 euro,

Reply:
Then I change it :fox_face:

Reply:
:wolf: :+1:

Reply:
1) I could let sumaform (or the user) create another variable

export PROXY_VERSION=3.1

but it seems too much for something that is probably not even used (we've been using "2.1" here for years without any consequence).

2) Or maybe I could just set it to "", which is "same version as server".

I'll try the latest option.

Reply:
ok it works with this

```
ebi-proxy:~ # cat config-answers.txt20171025-5889-yxg00i 
RHN_PARENT=ebi-server.tf.local
HTTP_PROXY=''
VERSION=''
(...)
```